internal/cmd/logout.go

```diff
@@ -42,11 +44,17 @@ $ infra logout --all --clear`,
 		RunE: func(cmd *cobra.Command, args []string) error {
 			if len(args) == 1 {
 				if all {
-					fmt.Fprintf(os.Stderr, "  Server is already specified. Ignoring flag [--all] and logging out of server %s.\n", args[0])
-					all = false
+					return fmt.Errorf("Argument [SERVER] and flag [--all] cannot be both specified.")
```

Comment:
Since `logout` is generally a safe thing to do – we can just default to logging out of all, even if the user specified a `SERVER` argument

Reply:
seems like that would complicate things again.

Reply:
discussed/resolved, keeping as is